ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -52,7 +52,7 @@
   /**
    * Google Mobile Ads Mediation Banner adapter.
    */
-  private MediationBannerAdapter adapter;
+  private WeakReference<MediationBannerAdapter> adapter;
```

Comment:
this seems doesn't help to fix missed onDestroy():
adLayout is leaked because the initial adapter holds it using to com.vungle.mediation.VungleBannerAdapter#getBannerView

mLayout should be moved back to MediationBannerAdapter, and VungleBannerAdapter#requestBannerAd could return adLayout to be stored in VungleInterstitialAdapter
this can be tested using the latest button on "corner case" screen:

<img width="513" alt="Screenshot 2020-08-06 at 12 47 33" src="https://user-images.githubusercontent.com/43134123/89518179-38e06700-d7e3-11ea-98b7-cc475ad24b31.png">

Reply:
As we synced offline, I tested bannerListener is the rest ref make 'adapter' leak. Have updated  bannerListener to weak ref.

Reply:
alright, After move adLayout back to VungleInterstitialAdapter, no 'adapter' leak find now.

Reply:
yes, here is leak: adLayout holds reference to VungleInterstitialAdapter
![image](https://user-images.githubusercontent.com/43134123/89650723-859f6d00-d8cb-11ea-84cd-7166f413a4e2.png)

Reply:
done.